workitem/workitem_repository.go

```diff
@@ -961,3 +962,38 @@ func (r *GormWorkItemRepository) GetCountsForIteration(ctx context.Context, itr
 	}
 	return countsMap, nil
 }
+
+// LoadByIteration returns the list of work items belongs to given iteration
+func (r *GormWorkItemRepository) LoadByIteration(ctx context.Context, iterationID uuid.UUID) ([]*WorkItem, error) {
+	defer goa.MeasureSince([]string{"goa", "db", "workitem", "loadByIteration"}, time.Now())
+	log.Info(nil, map[string]interface{}{
+		"itr_id": iterationID,
+	}, "Loading work items for iteration")
+
+	res := []WorkItemStorage{}
+	filter := fmt.Sprintf(`fields @> '{"system.iteration":"%s"}'`, iterationID)
```

Comment:
Use `workitem.SystemIteration` constant

Reply:
Yeah, fixed in https://github.com/fabric8-services/fabric8-wit/pull/1683/commits/129bdba992f8834a29072258d94969796a29121a